### DISCUSSION TITLE
Relation （or difference) between wry and tauri?

### DISCUSSION THREAD

**Discussion by fei4xu** [tauri-apps/wry#870] [General]
Hi,

In Wry introduction, it's "Cross-platform WebView library in Rust for Tauri" and it seems that it's maintained by tauri (the team of tauri).

-- I want to be clear, is tauri (tauri.app) currently using wry? or in the future?
(from https://github.com/tauri-apps/tauri/blob/dev/ARCHITECTURE.md#wry I can now confirm that tauri.app uses wry)

I also see a benchmark at https://tauri.app/v1/references/benchmarks, it seems that tauri (tauri.app) are competing with wry and for most benchmark items, tauri.app are winners.

-- if tauri.app actually uses wry, why tauri has better results in the benchmark in the end ?

I'm really confused.

**Comment by fei4xu** [comment]
As I read more, I realized that the benchmark is not intended to compare tauri to wry but electron. 

The comparison has two parts: tauri vs electron, and raw wry vs electon. In this way, it makes more sense to me.

**Comment by amrbashir** [comment]
Tauri indeed uses wry under the hood. The reason that wry does bad in the `Execution time` benchmarks is because github action `latest` runner has been upgraded to ubuntu 22.04 and it has a bug (specific to github action) that causes slow startup time for wry/tauri apps. We haven't been able to identify the bug so far nor we could reproduce it locally so for now we have downgraded the runner to `ubunutu 20.04` on tauri's benchmarks but we haven't done the same for wry yet.

**Reply by wusyong** [reply]
That's github action specific bug I think. I'm on PopOS 22.04 and everything still work as usual.
